docs/source/frameworks.rst

```diff
@@ -1,227 +1,238 @@
-
-.. autoclass:: bentoml.frameworks.lightgbm.LightGBMModelArtifact
-
-
-========
-FastText
-========
-
-Example Projects:
-
-* Text Classification - `Google Colab <https://colab.research.google.com/github/bentoml/gallery/blob/master/fasttext/text-classification/text-classification.ipynb>`__ / `Notebook Source <https://github.com/bentoml/gallery/blob/master/fast-text/text-classification/text-classification.ipynb>`__
-
-.. autoclass:: bentoml.frameworks.fasttext.FasttextModelArtifact
-
-===
-H2O
-===
-
-Example Projects:
-
-* Loan Default Prediction - `Google Colab <https://colab.research.google.com/github/bentoml/gallery/blob/master/h2o/loan-prediction/h2o-loan-prediction.ipynb>`__ / `Notebook Source <https://github.com/bentoml/gallery/blob/master/h2o/loan-prediction/h2o-loan-prediction.ipynb>`__
-* Prostate Cancer Prediction - `Google Colab <https://colab.research.google.com/github/bentoml/gallery/blob/master/h2o/prostate-cancer-classification/h2o-prostate-cancer-classification.ipynb>`__ / `Notebook Source <https://github.com/bentoml/gallery/blob/master/h2o/prostate-cancer-classification/h2o-prostate-cancer-classification.ipynb>`__
-
-.. autoclass:: bentoml.frameworks.h2o.H2oModelArtifact
-
-
-======
-CoreML
-======
-
-.. autoclass:: bentoml.frameworks.coreml.CoreMLModelArtifact
-
-
-====
-ONNX
-====
-
-Example Projects:
-
-* Image Classification with ResNet50 - `Google Colab <https://colab.research.google.com/github/bentoml/gallery/blob/master/onnx/resnet50/resnet50.ipynb>`__ / `Notebook Source <https://github.com/bentoml/gallery/blob/master/onnx/resnet50/resnet50.ipynb>`__
-
-.. autoclass:: bentoml.frameworks.onnx.OnnxModelArtifact
-
-
-=====
-Spacy
-=====
-
-.. autoclass:: bentoml.frameworks.spacy.SpacyModelArtifact
-
-
-============
-Transformers
-============
-
-.. autoclass:: bentoml.frameworks.transformers.TransformersModelArtifact
-
-
-===========
-Statsmodels
-===========
-
-For statsmodels, we recommend using PickleArtifact:
-
-.. autoclass:: bentoml.service.artifacts.common.PickleArtifact
-    :noindex:
-
-Example Projects:
-
-* Shampoo Sales Prediction -  `Google Colab <https://colab.research.google.com/github/bentoml/gallery/blob/master/statsmodels_holt/bentoml_statsmodels.ipynb>`__ / `Notebook Source <https://github.com/bentoml/gallery/blob/master/statsmodels_holt/bentoml_statsmodels.ipynb>`__
-
-
-=====
-Gluon
-=====
-
-.. autoclass:: bentoml.frameworks.gluon.GluonModelArtifact
-
-
-=================
-Pytorch Lightning
-=================
-
-.. autoclass:: bentoml.frameworks.pytorch.PytorchLightningModelArtifact
-
-
-=========
-Detectron
-=========
-
-.. autoclass:: bentoml.frameworks.detectron.DetectronModelArtifact
-
-
-.. spelling::
-
-    MLModel
-    tokenizer
-    AutoModelWithLMHead
-    nn
-    fasttext
-    onnx
-    Statsmodels
-    mxnet
-    env
-    GluonModelArtifact
-    JsonInput
-    nd
-    mx
-    asnumpy
-    svc
-    Pytorch
-    pytorch
-    detectron
-    Detectron
-    DetectionCheckpointer
-
+Frameworks

```

Comment:
These lines are unnecessarily marked as modified. I guess this file was saved with the Windows-style newline \r\n, right?